aiida/cmdline/commands/cmd_process.py

```diff
@@ -185,11 +185,33 @@ def process_call_root(processes):
 @click.option(
     '-m', '--max-depth', 'max_depth', type=int, default=None, help='Limit the number of levels to be printed.'
 )
+@click.option(
+    '-e', '--end', 'last', is_flag=True, default=False, help='Report for the most recent node in verdi process list -a'
```

Comment:
Don't think that `--end` is really intuitive. Maybe something like the following would be more intuitive
```suggestion
    '-M', '--most-recent-node', 'most_recent_node', is_flag=True, default=False, help='Select the most recently created process node.'
```

I chose `-M` because `-m` is already in use. I think it would make sense to also add this option to `verdi process show` and `verdi process status`. These also don't use `-M` yet

Reply:
I agree, I originally wanted `--last`, but the shorthand for that would be `-l`, which is ambiguous (l vs I),  and I associate this with `list`. Your suggestion is definitely better, though perhaps `--recent`/`-R` would be more intuitive? Or even `--last` and `-L`?

Reply:
I find `--most-recent-node` way more intuitive than `--recent` because the latter doesn't imply a single node, but could be a few nodes, e.g. all nodes from last 24 hours. I can see how one could object to the longer option name, but that is why there is the shorthand _and_ we have tab completion of options.